src/main/resources/assets/js/components/modals/AboutModalComponent.jsx

```diff
@@ -6,14 +6,6 @@ var BackboneMixin = require("../../mixins/BackboneMixin");
 var ModalComponent = require("../components/../ModalComponent");
```

Comment:
No, it isn't. I also thought this, but I was wrong.
See:
https://github.com/mesosphere/marathon/pull/1123#issuecomment-72574699